app/backend/src/couchers/errors.py

```diff
@@ -41,6 +41,8 @@
 LOGOUT_FAILED = "Couldn't log out."
 MISSING_BOTH_PASSWORDS = "You must supply at least the old or the new password."
 MISSING_PASSWORD = "The user has a password but the request did not provide one."
+PASSWORD_TOO_SHORT = "The password must be 8 or more characters long."
+INSECURE_PASSWORD = "The password is insecure. Please use one that is not easily guessable."
```

Comment:
Could you sort the file, please? (You seem to be on VS code, you can find "sort ascending" in the command menu")